### ISSUE TITLE
Quick Look in OS X

### ISSUE THREAD

**Issue by pgolebiowski** [typora/typora-issues#494]
It would be a nice feature if Typora could support Quick Look in OS X 😄 For me, extremely helpful to quickly preview files without opening them. Just pressing a space bar and voila!

**Comment by MilkTicc** [comment]
https://github.com/toland/qlmarkdown

**Comment by AlexPasternak** [comment]
Considering the very permissive distribution license for that component, this seems like a no-brainer. +1.

That said, my QuickLook looks fine already for .md files, what does it look like for you guys? Maybe it's TextMate taking care of it, because that's what the preview styling looks like. Definitely good enough to get a gist of the file.

![untitled](https://cloud.githubusercontent.com/assets/1819485/22332919/5df53014-e3a1-11e6-9611-602b9a846a07.png)

**Comment by pgolebiowski** [comment]
Such a solution does work, but (at least for me) being able to see the markdown rendered in current theme in Typora would be just awesome.

**Comment by kadmonj** [comment]
I haven't found yet a quick look plugin that can render math.
It would be really useful to see mmd notes with equations. If you have many files with many math expressions, browsing through them in finder using quick look is not natural.

a QL plugin with mathjax support would be a new thing in the landscape as far as I can tell

**Comment by Roderick-Jonsson** [comment]
Yes Please. I only have typora installed and can't quick look md files. This should really be included with typora.
Is there are chance that this is coming soon? @abnerlee

**Comment by abnerlee** [comment]
@Roderick-Jonsson Won't "coming soon" since it can be supported by https://github.com/toland/qlmarkdown

May consider this after v1.x release

**Comment by AlexPasternak** [comment]
Too bad. For now I have Quick Look MD previews anyway, because I believe I have Marked editor installed as part of Setapp, so you may want to look into that.

**Comment by vassudanagunta** [comment]
I'm with @abnerlee. Typora has more important priorities to get to first!

**Comment by pgolebiowski** [comment]
@vassudanagunta #NotFunny

**Comment by Wsine** [comment]
> @Roderick-Jonsson Won't "coming soon" since it can be already supported by https://github.com/toland/qlmarkdown
> 
> May consider this after v1.x release

The typora version comes to v1.7.6. Is it the time to consider this feature?

According to https://apple.stackexchange.com/a/120625, many markdown editors support this feature.

- [BBEdit](http://www.barebones.com/products/bbedit/)
- [TextMate](http://macromates.com/)
- [Mou](http://mouapp.com/)
- [Marked](http://markedapp.com/)
- [MarkdownLive](https://github.com/rentzsch/markdownlive)
- [Atom](https://github.com/atom/markdown-preview)
- [Marked 2](http://marked2app.com/)
- [MacDown](http://macdown.uranusjr.com/)
